platforms/__common.sh

```diff
@@ -129,7 +129,7 @@ getSdkFromImage() {
 	echo
 	docker run \
 		--rm \
-		-v $hostVolumeDir:$volumeContainerDir \
+		-v /$hostVolumeDir:$volumeContainerDir \
```

Comment:
@pauld-msft is the `/` a typo?

Reply:
@william-msft nope, this is to allow buildPlatformBinaries.sh to move tar files into the artifacts directory for php, node, and java when executed on windows machine. Without this change, it doesn't work locally for those languages.